src/LoaderController.res

```diff
@@ -105,6 +111,20 @@ let make = (~children, ~paymentMode, ~setIntegrateErrorError, ~logger) => {
     let appearance =
       optionsAppearance == CardTheme.defaultAppearance ? config.appearance : optionsAppearance
 
+    let sdkHandleConfirmPaymentProps = mergeJsons(
+      defaultButtonRules->toJson,
+      optionsDict
+      ->Js.Dict.get("sdkHandleConfirmPaymentProps")
+      ->Belt.Option.getWithDefault(Js.Json.null),
+    )
+
```

Comment:
see this happens when you collect the customization for the button separately why not take all the customizations at once

Reply:
Updated